ocaml/xapi/import.ml

```diff
@@ -1205,16 +1205,31 @@ module VGPU : HandlerTools = struct
       let vm = log_reraise
           ("Failed to find VGPU's VM: " ^ (Ref.string_of vgpu_record.API.vGPU_VM))
           (lookup vgpu_record.API.vGPU_VM) state.table in
-      let group = log_reraise
+      let group =
+        (* If we find the cross-pool migration key, attach the vgpu to the provided gpu_group... *)
+        if List.mem_assoc Constants.storage_migrate_vgpu_map_key vgpu_record.API.vGPU_other_config
+        then Ref.of_string (List.assoc Constants.storage_migrate_vgpu_map_key vgpu_record.API.vGPU_other_config)
```

Comment:
It will fail the compatibility check called during `migrate_send` and `assert_can_migrate` and raise an error there, before starting the `import`